doc-content/enterprise-only/employee-rostering/optashift-ER-running-jar-proc.adoc

```diff
@@ -1,6 +1,6 @@
 [id='optashift-ER-running-jar-proc']
 = Running the Employee Rostering starter application JAR file
-You can run the Employee Rostering starter application from a JAR file included in the *{PRODUCT} {PRODUCT_VERSION} Decision Services Quick Starts* download.
+You can run the Employee Rostering starter application from a JAR file included in the *{PRODUCT} {PRODUCT_VERSION_LONG} Kogito and OptaPlanner 8 Decision Services Quickstarts* download.
 
 .Prerequisites
 * You have downloaded and extracted the `kogito-1.5.0-decision-services-quickstarts.zip` file as described in <<er-deploy-prepare-proc>>.
```

Comment:
You need to substitude 
 kogito-1.5.0-decision-services-quickstarts
to
rhpam-7.11.0-decision-services-quickstarts and rhdm-7.11.0-decision-services-quickstarts accordingly